clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2022,6 +2035,22 @@ bool CompilerInvocation::ParseCodeGenArgs(CodeGenOptions &Opts, ArgList &Args,
       Diags.Report(diag::err_drv_invalid_value) << A->getAsString(Args) << Name;
   }
 
+  if (const Arg *const A = Args.getLastArg(OPT_fcf_branch_label_scheme_EQ)) {
```

Comment:
```suggestion
  if (const Arg *A = Args.getLastArg(OPT_fcf_branch_label_scheme_EQ)) {
```

For consistency with other site within this file.

Reply:
Fixed